webrender/src/display_list_flattener.rs

```diff
@@ -539,6 +543,17 @@ impl<'a> DisplayListFlattener<'a> {
             ),
         );
 
+        match clip_and_scroll_ids.clip_node_id {
```

Comment:
I think what you want to do here is to simply use the newly added clip node. So

```rust
self.pipeline_clip_chains.insert(
    iframe_pipeline_id, 
    self.id_to_index_mapper.get_node_index(info.clip_id)
);
```

Reply:
but that's the whole point of the PR - to take `clip_and_scroll_ids` into account